installer.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -o errexit #Stops the installer if ANY line of code returns a non-zero exit code.
```

Comment:
Did some research and yes set -e is the same as set -o errexit. set -e is actually just a pointer to set -o errexit. Since that is the case this pull request adds no value to the project, so I will be closing it.